frontend/lib/ui/dashboard/find_shortcut.dart

```diff
@@ -13,38 +13,38 @@ class FindShortcut extends ConsumerWidget {
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
-    final shouldActivateShortcut = AppRoutes.isAtDashboardPage(context);
+    final shouldMakeFiltersVisible =
+        AppRoutes.isDashboardPage(AppRoutes.uriFromContext(context));
```

Comment:
Just curious, do we want to bind ctrl+f to the search bar on each artefact page, or just in the dashboard page?

Reply:
ctrl+f does work on both pages now. This condition is to display the hidden filters on dashboard page